src/components/organisms/AdminVenueView/components/Spaces/Spaces.tsx

```diff
@@ -0,0 +1,236 @@
+    icon: "/assets/icons/icon-room-experience.svg",
+    template: VenueTemplate.zoomroom,
+  },
+  {
+    text: "External link",
+    icon: "/assets/icons/icon-room-externallink.svg",
+  },
+  {
+    text: "Map",
+    icon: "/assets/icons/icon-room-map.svg",
+    template: VenueTemplate.partymap,
+  },
+];
+
+export interface SpacesProps {
+  venue: Venue_v2;
+  onClickNext: () => void;
+}
+
+export const Spaces: React.FC<SpacesProps> = ({ venue, onClickNext }) => {
+  const [selectedRoom, setSelectedRoom] = useState<RoomData_v2>();
+  const [showRooms, setShowRooms] = useState<boolean>(false);
+  const [updatedRoom, setUpdatedRoom] = useState<RoomData_v2>({});
+
+  const [showAddRoom, setShowAddRoom] = useState<boolean>(false);
+  const [showAdvancedSettings, setShowAdvancedSettings] = useState<boolean>(
+    false
+  );
+
+  const history = useHistory();
+
+  const hasSelectedRoom = !!selectedRoom;
+  const numberOfRooms = venue.rooms?.length ?? 0;
+
+  const clearSelectedRoom = useCallback(() => {
+    setSelectedRoom(undefined);
+    setUpdatedRoom({});
+  }, []);
+
+  const updateRoomPosition = useCallback(async (position: Position) => {
+    if (!position) return;
+
+    setUpdatedRoom((room) => ({
+      ...room,
+      x_percent: position.left,
+      y_percent: position.top,
+    }));
+  }, []);
+
+  const updateRoomSize = useCallback(async (size: Dimensions) => {
+    if (!size) return;
+
+    setUpdatedRoom((room) => ({
+      ...room,
+      width_percent: size.width,
+      height_percent: size.height,
+    }));
+  }, []);
+
+  const selectedRoomIndex =
+    venue.rooms?.findIndex((room) => room === selectedRoom) ?? -1;
+
+  return (
+    <div className="spaces">
+      <div className="spaces__rooms">
+        {hasSelectedRoom ? (
+          <EditSpace
+            room={selectedRoom!}
+            updatedRoom={updatedRoom}
+            roomIndex={selectedRoomIndex}
+            onBackPress={clearSelectedRoom}
+            onDelete={clearSelectedRoom}
+            onEdit={clearSelectedRoom}
+          />
+        ) : (
+          <>
+            <div className="spaces__background">
+              <div className="spaces__title">Build your spaces</div>
+            </div>
+            <div>
+              <div
+                className="spaces__venue-rooms"
+                onClick={() => setShowAdvancedSettings(!showAdvancedSettings)}
+              >
+                <div>Map background</div>
+                <FontAwesomeIcon
+                  icon={!showAdvancedSettings ? faCaretRight : faCaretDown}
+                />{" "}
+              </div>
+              {showAdvancedSettings && (
+                <BackgroundSelect venueName={venue.name} mapBackground={""} />
+              )}
+            </div>
+
+            <div>
+              <div
+                className="spaces__venue-rooms"
+                onClick={() => setShowRooms(!showRooms)}
+              >
+                <div>Rooms {numberOfRooms}</div>
```

Comment:
swap `Rooms` and `{numberOfRooms}` because right now it looks like "Rooms 4" instead of "4 Rooms"
```suggestion
                <div>{numberOfRooms} Rooms</div>
```

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1583/commits/d2bc2049fe72c651d807803b6b568d74a74764f6